apps/cli/src/commands/build.ts

```diff
@@ -132,36 +132,74 @@ export default class BuildApplication extends Command {
         return info;
     }
 
+    // creates docker image tar
     private async exportImageTar(
         image: string,
-        tarPath: string,
+        path: string,
     ): Promise<void> {
-        // create container
-        const { stdout: cid } = await execa("docker", [
-            "container",
-            "create",
-            "--platform",
-            "linux/riscv64",
+        // delete tar if exists
+        if (fs.existsSync(path))
+            fs.unlinkSync(path);
```

Comment:
Do we need this? I already do a `fs.emptyDir(".sunodo")` in the beginning of the build process

Reply:
It was just to remove the `docker-image.tar` that I consider an intermediary.

But thinking about it, we could remove all except `./sunodo/image/` (used for `run`) and `.sunodo/image.ext2`  (used for `shell`).

Reply:
Currently `.sunodo` is cleared in the beginning of the build process

Reply:
Yes, and the build result is kept until the next build, and these other files are kept there, just using space.

So I think we could remove not just the `./sunodo/docker-image.tar` but also the `./sunoso/image.tar`.

Reply:
Yeah, I intentionally left the image.tar around mainly for debug reasons.

Reply:
So I prefer removing the `.sunodo/docker-image.tar` since it wouldn't help with debugging from the POV of the generated `image.ext2`.